extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -508,29 +545,55 @@ export class Visibility {
       this.setState_(state, visible, 0);
     }
 
-    const waitForContinuousTime = config[CONTINUOUS_TIME_MIN]
-        ? config[CONTINUOUS_TIME_MIN] - state[CONTINUOUS_TIME]
-        : Infinity;
-    const waitForTotalTime = config[TOTAL_TIME_MIN]
-        ? config[TOTAL_TIME_MIN] - state[TOTAL_VISIBLE_TIME]
-        : Infinity;
+    listener['state'] = state;
+
+    return this.isConditionMet_(state, config, triggerType);
+  }
+
+  /**
+   * @param {!Object} state
+   * @param {!Object} config
+   * @return {number}
+   * @private
+   */
+  computeTimeToWait_(state, config) {
+    const waitForContinuousTime =
```

Comment:
Please explain each formula and comments and indicate what are the default configuration values. Otherwise it's confusing. I'd expect `0` in the else clause.